.github/workflows/pull-request-conditionals.yaml

```diff
@@ -70,20 +71,65 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
+      # Add a custom filter for all packages that excludes any changes to IaC testing related config
+      # Noting that adding `predicate-quantifier` produces a false warning message about an unexpected input, despite still being evaluated by the action
+      - name: Check src paths - all
+        id: path-filter-all
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3
+        with:
+          predicate-quantifier: 'every'
+          filters: |
+            all:
+              - "**"
+              - "!.github/workflows/test-aks.yaml"
+              - "!.github/workflows/test-eks.yaml"
+              - "!.github/workflows/test-rke2.yaml"
+              - "!.github/bundles/**"
```

Comment:
Should this also include `!.github/test-infra/**` and `!tasks/iac.yaml`? (if so also would apply to the shim in the reverse)